src/group_impl.h

```diff
@@ -7,7 +7,7 @@
 #ifndef SECP256K1_GROUP_IMPL_H
 #define SECP256K1_GROUP_IMPL_H
 
-#include "field.h"
+#include "field_impl.h"
```

Comment:
So I had to make this change to get `precompute_ecmult_gen.c` to build.

I'm not sure what the rules on including *_impl files is.  Should `secp256k1_const_beta` (and `secp256k1_fe_one`) be moved from `field_impl.h` to `field.h` instead ?

Reply:
Well, at the moment there are no written rules. I believe the rules should look like described here: https://github.com/bitcoin-core/secp256k1/issues/1039#issuecomment-992692922 but we don't adhere to them everywhere. 

And yes, moving the constants to `field.h` would take us a step closer to this. So if this works, feel free to do it here but otherwise let's stick with including the `_impl.h` file until we have a clean solution to #1039.

Reply:
constants are moved.